net-p2p/retroshare/metadata.xml

```diff
@@ -32,6 +32,7 @@
 	</use>
 	<upstream>
 		<bugs-to>https://github.com/RetroShare/RetroShare/issues</bugs-to>
+		<!-- NOTE(setan): for core functionalities check cout RetroShare/libretroshare -->
```

Comment:
Shouldn't this be a note in the ebuild, not the metadata.xml? The metadata.xml is intended for programmatic usage, comments in this file aren't really useful except as comments to people who will later on be updating metadata.xml itself.

This comment looks like something that one would want to remember when updating the ebuild with new functionality. Am I missing something?